docs/Engineering/TechStack.mdx

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_label: "Technology stack"
+---
+
+import MaintainerTag from "@site/src/components/MaintainerTag";
+import ControlledDocBanner from "@site/src/components/ControlledDocBanner";
+
+
+<MaintainerTag maintainerEmails={["siddharth.krishna@openenergytransition.org","johannes.hampp@openenergytransition.org"]} />
+
+# Technology stack that we use
+
+Recommended, not enforced.
+Projects may deviate, especially in case of projects based on soft forks.
+
+
+* Version control:
+    * `git`
+    * Host repositories on: `github`
+    * Repositories should belong to [Open Energy Transition GitHub organization](https://github.com/open-energy-transition)
+    * No force pushes
+    * Use pull requests for all changes
+    * Merge branches instead of rebasing
+    * Squash and merge pull requests to keep the commit history clean
+* Continuous integration:
+    * For public repositories / open source projects: `github actions`
+    * For private repositories / internal projects: ?
```

Comment:
If they are lightweight, github actions could still be possible (what's our total private repo allowance?)